### PR TITLE
foreman_remote_execution 1.3.5 - foreman part [deb/1.15] 

### PR DESCRIPTION
**Pull request by iNecas**
For plugin updates, please indicate which repos this should be built into:

* [ ] Nightly
* [ ] 1.16
* [x] 1.15
* [ ] 1.14

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by iNecas** [comment]
I've upated this PR to bump to 1.3.5 to include http://projects.theforeman.org/issues/21242

File: plugins/ruby-foreman-remote-execution/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+ruby-foreman-remote-execution (1.3.4-1) stable; urgency=low
```
Comment (mmoll) [inline plugins/ruby-foreman-remote-execution/debian/changelog]: needs to be 1.3.5-1

File: plugins/ruby-foreman-remote-execution/foreman_remote_execution.rb [hunk 2]
```diff
@@ -1 +1 @@
-gem 'foreman_remote_execution', '1.3.3'
+gem 'foreman_remote_execution', '1.3.4'
```
Comment (mmoll) [inline plugins/ruby-foreman-remote-execution/foreman_remote_execution.rb]: needs to be 1.3.5

**Comment by ekohl** [comment]
Then the title of this PR should also be 1.3.5 :)

**Comment by iNecas** [comment]
Comments addressed

**Comment by mmoll** [comment]
merged, díky @iNecas!

**Pull request merged**